src/csv/mocks/csv-req-res.mock.ts

```diff
@@ -0,0 +1,26 @@
+import { CreateProductDto } from 'src/products/dto/create-product.dto';
+import { CreatedProductDto } from 'src/products/dto/created-product.dto';
+
+export const mockCSV = {
+  buffer: Buffer.from(__dirname + '../../../test/mock.csv'),
+} as Express.Multer.File;
+
+export const csvProdCreateRequest = {
+  csv: mockCSV,
+};
+
+export const csvProdCreatedResponse: CreatedProductDto[] = [
```

Comment:
Use anatine's zod-mock lib with generateMock(yourZodSchema) to create fake data for testing.